src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-weight: 400;
+  width: 200px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+
+  &__image {
+    margin-top: 32px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-description {
+    width: 166px;
+    height: 36px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-top: 40px;
+    color: $main-clr;
+  }
+
+  &__product-code {
+    height: 14px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $accent-clr;
+    margin: 4px 0 0 16px;
+    align-self: flex-start;
+  }
+
+  &__reviews {
+    width: 166px;
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 10px;
+    color: $main-clr;
+    margin-top: 16px;
+  }
+
+  &__price {
+    width: 166px;
+    height: 18px;
+    line-height: 18px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+
+    & :first-child {
+      font-size: 12px;
+      color: $accent-clr;
+    }
+
+    & :last-child {
+      font-weight: 700;
+      font-size: 16px;
+      color: $main-clr;
+    }
+  }
+
+  &__button {
+    margin: 16px 0;
+    width: 166px;
```

Comment:
still not fixed:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/db0edd66-9834-430e-bec9-174fb2f07213)